index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link  href="style.css" rel="stylesheet">
+   
+    <title>Zuhra Hashimi - Portfolio</title>
+</head>
+
+<body>
+    <div>
+    <section>
+        <header>
+            <div>
+            <nav class="topnav">
+            
+                <a href="#home" class="logo">Zuhra Hashimi</a>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </div>
+        </header>
+    </section></div>
```

Comment:
- [ ] Kindly remove the `header` in the `div`there's no need to put it into the `div`, delete even the `section`  because the HTML semantics help the browser to know witch section layout first. so your code should be like `<header>... </header>`  if you want to write  a clean code please read [html semantics](https://www.w3schools.com/html/html5_semantic_elements.asp)